mir_eval/separation.py

```diff
@@ -422,6 +703,33 @@ def _bss_source_crit(s_true, e_spat, e_interf, e_artif):
     return (sdr, sir, sar)
 
 
+def _bss_image_crit(s_true, e_spat, e_interf, e_artif):
+    """Measurement of the separation quality for a given image in terms of
+    filtered true source, spatial error, interference and artifacts.
+
+    Parameters
```

Comment:
You mean I should remove the docstrings for all underscore functions? Why not leave them since they don't hurt and could help others understand the code. I could remove the parameters though since they aren't described.